### PR TITLE
Kelsey Push Test

### PR DESCRIPTION
**Pull request by kahyde**
Testing first pull request =)

### CODE REVIEW

File: client/src/containers/home/index.js [hunk 1]
```diff
@@ -16,6 +16,7 @@ export default class Home extends React.Component {
     return (
       <div className="page-home">
         <h4>Hello world! This is an example Pull Request</h4>
+				<p>Kelsey Push Test</p>
```
Comment (kahyde) [inline client/src/containers/home/index.js]: Any idea why this gets indented? It's not like that on my local copy and looking back through some of my projects from yonks ago I've noticed weird indenting there as well...

Comment (ohyde) [inline client/src/containers/home/index.js]: What editor are you using? It's the difference between spacing and tabs. I tend to set my editor to default as spaces and convert anything that's a tab into a space for consistency.

Comment (ohyde) [inline client/src/containers/home/index.js]: Did you get the app running?

Comment (kahyde) [inline client/src/containers/home/index.js]: Just using atom on defaults. I'll look into it, thanks.

Nah, just wanted to check I could commit and do pull requests first.

Comment (ohyde) [inline client/src/containers/home/index.js]: Mission Successful!

Comment (lucasschejtman) [inline client/src/containers/home/index.js]: @ohyde maybe a task should be to setup [prettier](https://github.com/prettier/prettier)

Comment (ohyde) [inline client/src/containers/home/index.js]: I'll have a read tomorrow, looks good.

**Pull request merged**